pkg/notes/document/document.go

```diff
@@ -30,7 +30,6 @@ import (
 	"github.com/sirupsen/logrus"
 	"golang.org/x/text/cases"
 	"golang.org/x/text/language"
-	"sigs.k8s.io/promo-tools/v3/image"
```

Comment:
I need to update the constant in that library. I'll do that if the PR looks good and revert the change to this file.

Reply:
nvm. 

https://kubernetes.slack.com/archives/CCK68P2Q2/p1667850096184829

TL;DR we need to have multiple complex conversations before merging https://github.com/kubernetes-sigs/promo-tools/pull/669